roledb.go

```diff
@@ -408,6 +436,17 @@ type RoleGroupCache struct {
 	hasAll       bool
 }
 
+// GetGroupsRaw's results are not cached. The point is to get the current state of the db for the export
+func (x *RoleGroupCache) GetGroupsRaw() ([]RawAuthGroup, error) {
+	x.groupsLock.RLock()
+	groups, err := x.backend.GetGroupsRaw()
+	if err != nil {
+		return nil, err
+	}
+	x.groupsLock.Unlock()
+	return groups, nil
```

Comment:
As discussed, this should just be the one liner returning the result of the function.